### PR TITLE
feat: export single result or all results to zip file

### PR DESCRIPTION
**Pull request by mohamed-ahdidou-nw**

### CODE REVIEW

**Comment by Almouro** [comment]
Closes #50

**Comment by mohamed-ahdidou-nw** [comment]
Video record

https://user-images.githubusercontent.com/89012789/207092716-2ae5da63-7fbb-4f84-a98b-1af81844b713.mov

File: package.json [hunk 1]
```diff
@@ -31,16 +31,24 @@
   "homepage": "https://github.com/bamlab/android-performance-profiler#readme",
   "devDependencies": {
     "@types/jest": "^26.0.3",
+    "assert": "^2.0.0",
```
Comment (Almouro) [inline package.json:34]: seems like a lot of things got added here, which probably don't need to be? 🤔

File: packages/web-reporter-ui/ReporterView.tsx [hunk 2]
```diff
@@ -37,9 +40,15 @@ const Report = ({ results }: { results: TestCaseResult[] }) => {
...
   );
 
+  const saveResultsToZIP = () => {
+    exportRawDataToZIP(results);
+  };
+
   return (
     <>
-      <ReportSummary results={averagedResults} />
+      <Header saveToZIPCallBack={saveResultsToZIP} />
```
Comment (Almouro) [inline packages/web-reporter-ui/ReporterView.tsx:49]: realizing that maybe we can avoid having the header if we have `results.length === 1` what do you think?

File: package.json (hunk 1 above)
Comment (mohamed-ahdidou-nw) [reply package.json:34]: I'm not sure how those dependencies have been added,
I'll make sure to remove the unnecessary ones

**Review by Almouro — APPROVED** [review]

**Pull request merged**